.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
I would rather not add dependabot. It will actually add more work for me to manage the pull requests as a single maintainer.

Reply:
Removed.